src/gui/FileBrowser.cpp

```diff
@@ -656,14 +663,22 @@ void FileBrowserTreeWidget::contextMenuEvent(QContextMenuEvent * e )
 		patternEditorHeader->setDisabled(true);
 		contextMenu.addAction(patternEditorHeader);
 		contextMenu.addActions( getContextActions(file, false) );
-
-		// We should only show the menu if it contains items
-		if (!contextMenu.isEmpty()) { contextMenu.exec( e->globalPos() ); }
+		break;
+	}
+	case TypeDirectoryItem: {
+		auto dir = dynamic_cast<Directory*>(item);
+		QString dirname = dir->fullName();
+		contextMenu.addAction(QIcon(embed::getIconPixmap("folder")), tr("Open in %1").arg(fileManager), [=, this] {
+			auto _dirname = dirname;
+			FileRevealer::openDir(QFileInfo(_dirname));
```

Comment:
I think the implicit conversation is OK here, it makes the code cleaner.

```suggestion
		contextMenu.addAction(QIcon(embed::getIconPixmap("folder")), tr("Open in %1").arg(fileManager), [=, this] {
			FileRevealer::openDir(dir->fullName());
		});
```